Test fails regularly: test_mail.py

"Is flaky" -Will

Error logs:

FAIL: test_messages_per_second (tests.test_mail.BulkEmailTests.test_messages_per_second)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/tests/test_mail.py", line 246, in test_messages_per_second
    self.assertAlmostEqual((bob.sent_time - alex.sent_time).total_seconds(), 0.1, places=2)
AssertionError: 0.105753 != 0.1 within 2 places (0.005752999999999994 difference)

--- a/server/tests/test_mail.py
+++ b/server/tests/test_mail.py
@@ -243,5 +243,5 @@ class BulkEmailTests(TestCase):
         send_bulk_email(task)
 
         (alex, bob, chris) = task.recipients.all()
-        self.assertAlmostEqual((bob.sent_time - alex.sent_time).total_seconds(), 0.1, places=2)
-        self.assertAlmostEqual((chris.sent_time - bob.sent_time).total_seconds(), 0.1, places=2)
+        self.assertAlmostEqual((bob.sent_time - alex.sent_time).total_seconds(), 0.1, delta=0.01)
+        self.assertAlmostEqual((chris.sent_time - bob.sent_time).total_seconds(), 0.1, delta=0.01)

